tests/integration/test_elle.py

```diff
@@ -0,0 +1,297 @@
+"""
+This file is part of RedisRaft.
+
+Copyright (c) 2020-2021 Redis Ltd.
+
+RedisRaft is licensed under the Redis Source Available License (RSAL).
+"""
+import time
+import pytest as pytest
+
+from .sandbox import ElleWorker, Elle, key_hash_slot
+
+
+@pytest.mark.skipif("not config.getoption('elle_threads')")
+@pytest.mark.elle_test()
+def test_elle_sanity(cluster_factory):
+    cluster1 = cluster_factory().create(3, raft_args={
+        'sharding': 'yes',
+        'external-sharding': 'yes'})
+    time.sleep(5)
+
+
+#@pytest.mark.skipif("not config.getoption('elle_threads')")
+#@pytest.mark.elle_test()
```

Comment:
Please fix this mark. It requires `@pytest.mark.elle_test()` right?

Reply:
no, as this is the "manual" test, the mark are only needed for the background/automatic tests.  we might include the skipif though.

Reply:
ok, the skipif should be there.  but not the mark